permissions/plugin-upload-pgyer-plugin.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "upload-pgyer"
+paths:
+- "ren/helloworld/upload-pgyer"
```

Comment:
The group ID part of this does not appear to resolve to a valid domain name. Not a blocker, but I expect there's a better choice, e.g. your service's domain name in reverse?

Reply:
helloworld.ren, this is my domain name